### PR TITLE
Add new prng class. Remove old prng recommendations.

### LINKED ISSUE #20: Add documentation for new rng req's

### PR DESCRIPTION
**Pull request by ryley-o**
closes #20. Only merge after we reach consensus on new Random class.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,102 +51,87 @@ function genTokenData(projectNum) {
...
+        a = b ^ (b >>> 9);
+        b = (c + (c << 3)) | 0;
+        c = (c << 21) | (c >>> 11);
+        c = (c + t) | 0;
+        return (t >>> 0) / 4294967296;
+      };
+    };
+    this.prngA = new sfc32(tokenData.hash.substr(2, 32));
+    this.prngB = new sfc32(tokenData.hash.substr(34, 32));
+    for (let i = 0; i < 5e5; i += 2) {
```
Comment (jakerockland) [inline README.md:80]: Shouldn't this be either `let i = 0; i < 1e6; i += 2` _or_ `let i = 0; i < 5e5; i += 1` but not `let i = 0; i < 5e5; i += 2`?

File: README.md [hunk 2]
```diff
@@ -51,102 +51,87 @@ function genTokenData(projectNum) {
...
+        d = (d + 1) | 0;
+        a = b ^ (b >>> 9);
+        b = (c + (c << 3)) | 0;
+        c = (c << 21) | (c >>> 11);
+        c = (c + t) | 0;
+        return (t >>> 0) / 4294967296;
+      };
+    };
+    this.prngA = new sfc32(tokenData.hash.substr(2, 32));
+    this.prngB = new sfc32(tokenData.hash.substr(34, 32));
```
Comment (jakerockland) [inline README.md:79]: Nit: could you document w/ comments that this takes the first half and second half of the tokenData.hash?

**Review by jakerockland — CHANGES_REQUESTED** [review]
Only one real piece here that I think needs a minor update, otherwise this looks fantastic :)

File: README.md (hunk 1 above)
Comment (ryley-o) [reply README.md:80]: ah, yes definitely! updating...

**Pull request merged**